### PR TITLE
rework galley dashboard to include new metrics

### PR DESCRIPTION
**Pull request by tangiel**

### CODE REVIEW

**Comment by tangiel** [comment]
We are also revisiting the dashboard test issue. I think my plan is to add them all to the query filter with a note that some of them should be removed if we can get traffic sent to Galley for the dashboard test.

**Comment by tangiel** [comment]
@costinm please review for approval in install/ and tests/

File: install/kubernetes/helm/subcharts/grafana/dashboards/galley-dashboard.json [hunk 1]
```diff
@@ -48,260 +48,1219 @@
...
+            "seriesOverrides": [],
+            "spaceLength": 10,
+            "stack": false,
+            "steppedLine": false,
+            "targets": [
+                {
+                    "expr": "sum by (typeURL) (galley_runtime_state_type_instances_total)",
+                    "format": "time_series",
+                    "intervalFactor": 1,
+                    "legendFormat": "",
```
Comment (douglas-reid) [inline install/kubernetes/helm/subcharts/grafana/dashboards/galley-dashboard.json]: do you want to have a legend format here?  even something like: "{{ typeURL }}" ?

**Pull request merged**